### ISSUE TITLE
Use of `<i>`

### ISSUE THREAD

**Issue by acabal** [drgrigg/wilkie-collins_no-name#5]
I see on `between-the-scenes-2.xhtml` line 145 there's an `<i>` tag styling the `<time>` element.  But, can we be more clever and do a selector like `[epub|type~="z3998:diary-entry"] time{ font-style: italic; }`?  (I don't know if *every* `<time>` element in a diary entry requires this styling; if not then this catchall selector might not be appropriate after all.)

I also see some unsemantic `<i>` uses. In the same file for example there is `the <i>Times</i>` which should have an `epub:type`; and `<i>both</i>` which should be `<em>both</em>`. You should do a global find for `<i>` to see where it should be replaced with `<em>` or otherwise get some semantic inflection.

**Comment by drgrigg** [comment]
(1) I don't think this is wise. There **are** other `<time>` entries which shouldn't be in italic in this file. I'm using italics for the entry date at the head of each entry, but they shouldn't be used for date references in the body of the entry.

(2) Fixed `<i>Times</i>` in this file with se:name.publication.newspaper. An oversight on my part.

Looking for other <i> references I'm a bit dubious as to whether I should add semantics (or just what semantics to add). Happy to have your advice.

For example, Wragge references a particular **heading** in his reference books ("Skins to Jump Into"). I've italicised this to match the scans. 

Or Mrs Wragge refers to **items in an advertisement** ("Try Finch's feeding bottle for Infants") etc. 

I've italicised **extracts from newspapers**, but don't know if I should add semantic tags to these.

The only other ones I can find are where Mrs Lecount is **dictating** to Noel Vanstone. Again, I can't work out if such instances should have semantics.

**Comment by acabal** [comment]
I ran a check and the selector `[epub|type~="z3998:diary-entry"] > p time` matches just the `<time>` elements we're interested in, so I added that selector and removed those `<i>` tags.

When you say you italicized extracts from newspapers, were they italicized in the scans to begin with?

**Comment by drgrigg** [comment]
Ah, I see that I've misread part of your email of 8 August, which I took on board thinking that newspaper extracts needed to be italicised (you were just referring to references to the **name** of the journal, I now see). I'll fix these.

When you say "I added that selector and removed those `<i>` tags" I don't see these changes in the repository here. Have you cloned the repository to Standard Ebooks?

**Comment by acabal** [comment]
Yes, the repo is cloned to the SE organization and the ebook has been released. If you make updates, you can apply them to the main repo with a pull request.

Great work as always, thanks!

**Comment by drgrigg** [comment]
Oh, great! Nice to see it up on your site.

There are still a couple of fixes which need to be applied to do with italics, but I’m still a novice when it comes to Git/GitHub and would welcome your advice about how to properly do a pull request.

I cloned a copy of the SE repository to my computer and created a (local) branch called Updates, to which I’ve applied the fixes. But I’m now at sea as to how to push those changes back up to the server. Did I need to create the branch on the server instead? Help! I’m nervous about doing something fatal now that the book is live.

Also, obviously now that I’m done with “No Name” I’m interested in looking at another book. Is “The Woman in White” still up for grabs? If not, or if the person working on “The Moonstone” would still like to work on it, then I’d be quite happy to look at Lytton Strachey’s “Eminent Victorians”.


Regards,

David

**Comment by acabal** [comment]
You can read this guide about pull requests: https://yangsu.github.io/pull-request-tutorial/

Don't worry too much about messing something up, I have to approve the pull requests before any merging actually happens.

Woman in White is still available, Eminent Victorians would also be a good production. Send a note to the mailing list whenever you're ready.

**Comment by drgrigg** [comment]
Thanks, the tutorial looks good.

But in following it closely I hit the same error as I did when trying to follow the example in “Pro Git”.  Here’s my series of commands I used after cloning the SE repository, just stepping through the tutorial:

git pull origin master

git checkout -b david_updates

git push origin david_updates


This is what I get when I try to push the branch.

remote: Permission to standardebooks/wilkie-collins_no-name.git denied to drgrigg.
fatal: unable to access 'https://github.com/standardebooks/wilkie-collins_no-name.git/': The requested URL returned error: 403


Is there a log-in/permissions command I've missed somewhere? Sorry to be a nuisance, but once I've figured this out I'll be all the better next time!

Regards,

David

**Comment by acabal** [comment]
You can't push to a Github repo you don't own. You have to create a new Github repo that clones the SE one, push your changes to that, then use the Github interface to create a pull request.
